h/api/search/test/config_test.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+import itertools
+import re
+import urllib
+
+from h.api.search.config import ANNOTATION_ANALYSIS
+
+
+def test_strip_scheme_char_filter():
```

Comment:
I presume the purpose of this filter is to remove tokens like "http(s)" that would otherwise appear in almost every URI?